tests/test_deploy_reqs.py

```diff
@@ -6,23 +6,42 @@
 import unittest
 import os
 import tempfile
-from mock import Mock
+from mock import Mock, patch
+from click.testing import CliRunner
 
 from shub import deploy_reqs
 
 
+@patch('shub.deploy_reqs.utils.build_and_deploy_egg')
 class TestDeployReqs(unittest.TestCase):
-    def test_can_decompress_downloaded_packages_and_call_deploy_reqs(self):
+    VALID_APIKEY = '1234'
+
+    def setUp(self):
+        self.runner = CliRunner()
+        os.environ['SHUB_APIKEY'] = self.VALID_APIKEY
```

Comment:
Hm, I'm not keen on this line -- side effects for the tests at process level.
It will be better with a context manager that restores the original value or unsets it, appropriately.